lib/src/parsers/nodes/security-header-parser.ts

```diff
@@ -0,0 +1,32 @@
+import { PropertyDeclaration } from "ts-simple-ast";
+import { Locatable } from "../../models/locatable";
+import { SecurityHeaderNode } from "../../models/nodes";
+import {
+  extractJsDocCommentLocatable,
+  extractPropertyNameLocatable
+} from "../utilities/parser-utility";
+import { parseType } from "../utilities/type-parser";
+
+/**
+ * Parse a `@securityHeader` decorated method.
+ *
+ * @param a field declaration
+ */
+export function parseSecurityHeader(
+  property: PropertyDeclaration
+): Locatable<SecurityHeaderNode> {
+  const decorator = property.getDecoratorOrThrow("securityHeader");
+
+  const name = extractPropertyNameLocatable(property);
+  const description = extractJsDocCommentLocatable(property);
+  const type = parseType(property.getTypeNodeOrThrow());
```

Comment:
Is there a verifier for the security header type? Allowed values should be numbers, strings or number/string literals

Reply:
Good point. Shouldn't it be only `string`?